{{ cookiecutter.repo_name }}/docs/aqa_frameworks/GDS/aqa_plan.md

```diff
@@ -0,0 +1,8 @@
+# Analytical quality assurance plan
+
+This analytical quality assurance (AQA) plan outlines our implementation of the [Aqua Book](
+https://www.gov.uk/government/publications/the-aqua-book-guidance-on-producing-quality-analysis-for-government) for
```

Comment:
2015, is this the most recent version?

Reply:
Yes it is Although I've added some additional related [resources](https://www.gov.uk/government/collections/aqua-book-resources) as well in commit [`e48b054 `](https://github.com/ukgovdatascience/govcookiecutter/pull/6/commits/e48b0546bd490fb6cedc5e16aa4436ce63c91e8e)!

Marking as **Resolved**.